### PR TITLE
Store: Add expanded card view for product reviews

### LINKED ISSUE #17048: Store: Product Reviews: Add UI for editing a single review
From the reviews list, it should be possible to select a 'review' for more editing options. See the `Review expanded` screen from #12989.

This should provide options for approving,  spamming, and trashing. It should also provide a link to #17047 for adding a reply to a review.

### PR DESCRIPTION
**Pull request by justinshreve**
This PR depends on #17969. It adds the 'expanded'/open view for a product review, based off of the design from #12989.

This PR provides a 'read only' view, so the action links (approve/trash/spam) do not work yet, but they do provide the correct status. It also doesn't include replies yet.

screen shot 2017-09-11 at 1 44 56 pm

screen shot 2017-09-11 at 1 45 12 pm

Closes #17048.

To Test:
* You must use https://github.com/woocommerce/wc-api-dev/tree/add/all-product-reviews-endpoint
* Make sure you have a few reviews on your site. You can import the demo content, or post a few.
* Load up this branch.
* Go to http://calypso.localhost:3000/store/reviews/:site
* Select a review to toggle it's open state.
* Make sure that all fields show properly.
* Select a second review to verify that you can open multiple reviews at once.
* Toggle the review closed to make sure that works.

### CODE REVIEW

**Comment by allendav** [comment]
We seem to lose linebreaks? Anyway to preserve those?

I.e. in wp-admin:

screen shot 2017-09-12 at 3 21 13 pm

But in calypso:

screen shot 2017-09-12 at 3 21 24 pm

**Comment by allendav** [comment]
"Spammed" seems odd and may be difficult to translate well. How about "Marked as Spam" instead?'

screen shot 2017-09-12 at 3 29 53 pm

File: client/extensions/woocommerce/app/reviews/review-actions-bar.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+const ReviewActionsBar = ( { review, currentStatus, translate } ) => {
```
Comment (allendav) [inline client/extensions/woocommerce/app/reviews/review-actions-bar.js]: It's not clear the difference between review.status and currentStatus. Maybe a comment?

File: client/extensions/woocommerce/app/reviews/review-card.js [hunk 2]
```diff
@@ -29,43 +33,141 @@ class ReviewCard extends Component {
...
+					</div>
+					<div className="reviews__date">{ humanDate( review.date_created_gmt + 'Z' ) }</div>
+				</div>
+				<AutoDirection>
+					<div className="reviews__content">
+						{ decodeEntities( stripHTML( review.review ) ) }
+					</div>
+				</AutoDirection>
+				<div className="reviews__rating">
+					<Rating rating={ ( review.rating / 5 ) * 100 } />
```
Comment (allendav) [inline client/extensions/woocommerce/app/reviews/review-card.js]: Nit. Old developer here. If you move the 100 to the left, multiply it by review.rating and then divide by 5 you can ditch the parens. If you feel especially cray cray, just multiply review.rating by 20. :)

File: client/extensions/woocommerce/app/reviews/review-card.js [hunk 3]
```diff
@@ -29,43 +33,141 @@ class ReviewCard extends Component {
...
 						<div className="reviews__date">{ humanDate( review.date_created_gmt + 'Z' ) }</div>
 					</div>
-					<AutoDirection>
-						<div className="reviews__content">
-							{ decodeEntities( stripHTML( review.review ) ) }
+
+					{ review.verified && (
+						<div className="reviews__verified-label">
+							<Gridicon icon="checkmark-circle" size={ 18 } />
+							<span>{ translate( 'Verified product owner' ) }</span>
```
Comment (allendav) [inline client/extensions/woocommerce/app/reviews/review-card.js]: Nit.  Could be shorter - i.e. "Product owner" or "Verified buyer" or "Bought product"

Comment (justinshreve) [reply client/extensions/woocommerce/app/reviews/review-card.js]: Agreed it's a bit long. I'll go with `Verified buyer` for now, unless @kellychoffman or @jameskoster have any opinions on it.

**Pull request merged**